src/en/TravisTranslations.lua

```diff
@@ -0,0 +1,226 @@
+-- {"id":4302,"ver":"1.0.10","libVer":"1.0.0","author":"MechTechnology"}
+
+local baseURL = "https://travistranslations.com"
+
+-- Filter Keys & Values
+local STATUS_FILTER = 2
+local STATUS_VALUES = { "All", "Ongoing", "Completed" }
+local ORDER_BY_FILTER = 3
+local ORDER_BY_VALUES = { "Latest Update", "Recently Added", "Popular", "Rating", "A-Z" }
+local ORDER_BY_TERMS = { "update", "new", "popular", "rating", "az" }
+local TYPE_FILTER = 4
+local TYPE_VALUES = { 
+	"Light Novel", 
+	"Novel", 
+	"Original Novel", 
+	"Web Novel" 
+}
+local LANGUAGE_FILTER = 8
+local LANGUAGE_VALUES = { 
+	"Chinese", 
+	"Japanese",
+	"Korean" 
+}
+local GENRE_FILTER = 50
+local GENRE_VALUES = { 
+	"Action",
+	"Adult",
+	"Adventure",
+	"BL",
+	"Comedy",
+	"Drama",
+	"Ecchi",
+	"Fantasy",
+	"Harem",
+	"Historical",
+	"Horror",
+	"Josei",
+	"Martial Arts",
+	"Mature",
+	"Mecha",
+	"Mystery",
+	"Psychological",
+	"Romance",
+	"Romance",
+	"Sci-Fi",
+	"Seinen",
+	"Shoujo",
+	"Shoujo Ai",
+	"Shounen",
+	"Shounen Ai",
+	"Slice of Life",
+	"Smut",
+	"Sports",
+	"Supernatural",
+	"Tragedy",
+	"Xianxia",
+	"Xuanhuan",
+	"Yaoi",
+	"Yuri",
+ }
+
+local searchFilters = {
+	DropdownFilter(ORDER_BY_FILTER, "Order by", ORDER_BY_VALUES),
+	DropdownFilter(STATUS_FILTER, "Status", STATUS_VALUES),
+	FilterGroup("Type", map(TYPE_VALUES, function(v,i) 
+		local KEY_ID = TYPE_FILTER + i
+		return CheckboxFilter(KEY_ID, v)
+	end)),
+	FilterGroup("Language", map(LANGUAGE_VALUES, function(v,i) 
+		local KEY_ID = LANGUAGE_FILTER + i
+		return CheckboxFilter(KEY_ID, v)
+	end)),
+	FilterGroup("Genre", map(GENRE_VALUES, function(v,i) 
+		local KEY_ID = GENRE_FILTER + i
+		return CheckboxFilter(KEY_ID, v)
+	end))
+}
+
+local encode = Require("url").encode
+
+local text = function(v)
+	return v:text()
+end
+
+local function shrinkURL(url)
+	return url:gsub("^.-travistranslations%.com", "")
+end
+
+local function expandURL(url)
+	return baseURL .. url
+end
+
+local function getPassage(chapterURL)
+	local content = GETDocument(expandURL(chapterURL)):selectFirst("main#primary")
+	-- Removes the title and makes it an H1 for consistant custom CSS.
+	local title = content:selectFirst("h2"):text()
+	local chap = content:selectFirst(".reader-content")
```

Comment:
Again might reuse content instead of new variable.

Reply:
Yea I changed that as well.